src/measure_theory/integral/circle_transform.lean

```diff
@@ -174,4 +173,308 @@ begin
+  simp only [inv_pos, absolute_value.pos_iff],
+  apply sub_ne_zero.2 (circle_map_ne_mem_ball w.2 y), },
+  let e := (∥ r ∥)⁻¹ * (ε/2),
+  have he : 0 < e, by {simp_rw e, apply mul_pos (inv_pos.2 hr) (div_pos hε two_pos) },
+  obtain ⟨a, ha⟩ := (hlim e he),
+  use a,
+  intros b hb,
+  simp_rw [deriv_circle_map, dist_eq_norm, ← mul_sub] at *,
+  have hg : ∥ (2 * π * I : ℂ)⁻¹ * (circle_map 0 R y * I *
+  ((circle_map z R y - ↑w)⁻¹ * (f (circle_map z R y) - F b (circle_map z R y))))∥ =
+  ∥ (2 * π * I : ℂ)⁻¹ * circle_map 0 R y * I * ((circle_map z R y - ↑w)⁻¹) ∥ *
+  ∥ (f (z + ↑R * exp (↑y * I)) - F b (z + ↑R * exp (↑y * I)))∥,
+  by {simp only [circle_map, abs_of_real, abs_exp_of_real_mul_I, mul_one, abs_I, abs_two,
+    norm_eq_abs, mul_inv_rev, inv_I, zero_add, one_mul, absolute_value.map_neg,
+    absolute_value.map_mul, map_inv₀],
+    ring,},
+  simp_rw [hg, ← r],
+  have hab := (ha b hb) (z + ↑R * exp (↑y * I)) (circle_map_mem_sphere z hR.le y),
+  simp only [abs_of_real, abs_exp_of_real_mul_I, add_sub_cancel',
+  mul_one, abs_mul, norm_eq_abs] at hab,
+  apply lt_trans (mul_lt_mul_of_pos_left hab hr),
+  simp_rw [e, div_eq_inv_mul, ← mul_assoc, mul_inv_cancel (ne_of_gt hr)],
+  simp only [one_mul, mul_lt_iff_lt_one_left, inv_eq_one_div],
+  linarith,
+end
+
+lemma circle_transform_of_uniform_exists_bounding_function {R : ℝ} {F : ℕ → ℂ → ℂ} (hR : 0 < R)
+  (f : ℂ → ℂ) (z : ℂ) (w : ball z R) (F_cts : ∀ n, continuous_on (F n) (sphere z R))
+  (hlim : tendsto_uniformly_on F f filter.at_top (sphere z R) ):
+  ∃ (bound : ℝ → ℝ), ((∀ n, ∀ᵐ r ∂(volume.restrict (Ioc 0 (2*π))),
+  ∥(circle_transform R z w (F n)) r∥ ≤ bound r)
+  ∧ integrable bound (volume.restrict (Ioc 0 (2*π)))) :=
+begin
+  have f_cont : continuous_on f (sphere z R) ,
+  by {apply tendsto_uniformly_on.continuous_on hlim,
+  simp only [F_cts, eventually_at_top, implies_true_iff, exists_const],},
+  simp only [metric.tendsto_uniformly_on_iff, gt_iff_lt, ge_iff_le, eventually_at_top] at hlim,
+  obtain ⟨a, ha⟩ := (hlim 1 (zero_lt_one)),
+  set bound : ℝ → ℝ := λ θ, (∑ i in finset.range (a+1) ,
+  complex.abs ((circle_transform R z w (F i)) θ))
+  + complex.abs ((circle_transform R z w (λ x, 1)) θ) +
+  complex.abs ((circle_transform R z w f) θ),
+  refine ⟨bound, _⟩,
+  split,
+  intro n,
+  rw [ae_restrict_iff'],
+  apply eventually_of_forall,
+  intros y hyl,
+  by_cases (n ≤ a),
+  simp_rw bound,
+  have hnn : n ∈ finset.range(a + 1), by {simp only [finset.mem_range], linarith},
+  have := finset.add_sum_erase (finset.range (a + 1))
+  (λ i , complex.abs ((circle_transform R z w (F i)) y)) hnn,
+  simp only [and_imp, mem_Ioc, finset.mem_range, mem_sphere_iff_norm, norm_eq_abs] at *,
+  simp_rw [←this, add_assoc, le_add_iff_nonneg_right],
+  apply add_nonneg,
+  apply (finset.sum_nonneg ),
+  intros a b,
+  apply absolute_value.nonneg,
+  apply add_nonneg,
+  apply absolute_value.nonneg,
+  apply absolute_value.nonneg,
```

Comment:
Looks like you didn't enclose subgoals in braces here. And can't you use the `positivity` tactic?